src/main/java/ch/njol/skript/effects/EffDrop.java

```diff
@@ -101,6 +101,20 @@ public void execute(Event e) {
 		}
 	}
 
+	private boolean isAir(ItemStack is){
```

Comment:
```suggestion
	private boolean isAir(ItemStack is) {
```

same for others, such as `if (..)`

Reply:
Also same for comments: `// Comment` instead of `//Comment`